tests/unit/lms/services/user_test.py

```diff
@@ -14,7 +14,7 @@ class TestUserService:
     def test_upsert_user(self, service, lti_user, db_session):
         user = service.upsert_user(lti_user)
 
-        saved_user = db_session.query(User).one()
+        saved_user = db_session.query(User).order_by(User.id.desc()).first()
```

Comment:
pyramid_request now creates an user (and pyramid_request is bring in by `lti_user`.